<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_sg90</class>
 <widget class="QWidget" name="Page_sg90">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sg90_Btn_return">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
   <property name="icon">
    <iconset resource="Btn_Icon.qrc">
     <normaloff>:/Btn_Icon/Icon/return.png</normaloff>:/Btn_Icon/Icon/return.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="sg90_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前角度：</string>
   </property>
  </widget>
  <widget class="QSlider" name="sg90_Slider">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Btn_Icon.qrc"/>
 </resources>
 <connections/>
</ui>
